client/packages/coldchain/src/Mobile/Monitoring/MobileTemperatureChart.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { TemperatureChart } from '../../common/Monitoring';
+import { Card } from '@common/components';
+
+export const MobileTemperatureChart = () => {
+  return (
+    <Card
+      sx={{
+        m: 1,
+        width: '100%',
+        border: '1px solid',
+        borderColor: '#eee',
+        borderRadius: 4,
+      }}
+    >
+      <TemperatureChart />
+    </Card>
```

Comment:
Wrapped it in a Card to follow the design around most components in mobile view.